### PR TITLE
Cookies API on webview.

### PR DESCRIPTION
**Pull request by janRucka**
Basically the same as chrome API but this one is working for webview:
https://developer.chrome.com/extensions/cookies

Documentation:
https://github.com/nwjs/nw.js/pull/5750

Our project:
https://groups.google.com/forum/#!topic/nwjs-general/TmZUq-JHIuQ

### CODE REVIEW

File: chrome/browser/extensions/api/cookies/cookies_api.cc [hunk 1]
```diff
@@ -207,25 +225,25 @@ CookiesGetFunction::CookiesGetFunction() {
...
+
+bool CookiesGetFunction::SetStoreBrowserContext() {
+  return extensions::SetStoreBrowserContext(this, parsed_args_, 
+    store_browser_context_);
+}
+
 bool CookiesGetFunction::RunAsync() {
-  parsed_args_ = Get::Params::Create(*args_);
+  parsed_args_ = GetParsedArgs();
   EXTENSION_FUNCTION_VALIDATE(parsed_args_.get());
```
Comment (rogerwang) [inline chrome/browser/extensions/api/cookies/cookies_api.cc]: Why is this changed, while the body of GetParsedArgs() and the line being replaced looks identical.

Comment (rogerwang) [inline chrome/browser/extensions/api/cookies/cookies_api.cc]: I see. That's because you're overriding this in the child class...

**Comment by janRucka** [comment]
Ok I created new pull-request (it is completely different so it seemed better idea than to edit this one) which does it in the way you propose. I have to admit it is much less code:
https://github.com/nwjs/chromium.src/pull/75
